.gitignore

```diff
@@ -41,3 +41,5 @@
 
 # Packaged module.
 *.tgz
+yarn.lock
+package-lock.json
```

Comment:
Why? Why are you changing our project settings? We do want to have exact NPM dep versions specified.